apps/game_backend/lib/game_backend/users/user.ex

```diff
@@ -38,6 +40,8 @@ defmodule GameBackend.Users.User do
       :game_id,
       :username,
       :last_afk_reward_claim,
+      :last_daily_reward_claim,
+      :last_daily_reward_claim_type,
```

Comment:
What does this mean?

Reply:
It's to know when a user claimed their last daily reward.